include/ddc/chunk.hpp

```diff
@@ -108,7 +120,9 @@ class Chunk<ElementType, DiscreteDomain<DDims...>>
      */
     Chunk& operator=(Chunk&& other)
```

Comment:
we should check for self-assignment, shouldn't we?

Reply:
I don't know, what should we expect from `a = std::move(a)` ?

Reply:
a noop, no ?

Reply:
https://cplusplus.github.io/LWG/issue2839 not that clear actually